README.md

```diff
@@ -58,31 +30,33 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Crypto Trade-off <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[Crypto Trade-off]** is a web-app that displays List crypto currency exchange site and their ranks.
 
 ## 🛠 Built With <a name="built-with"></a>
 
+- React
+- API
+
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/CSS">CSS</a></li>
+    <li><a href="https://react.org/">React</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="http://localhost:3000/">Crypto Trade-off</a></li>
   </ul>
 </details>
 
```

Comment:
- [ ] You did not use the Postgresql database for this project, it would be nice if you removed the information that you used Postgresql as a database. 👍 

![image](https://user-images.githubusercontent.com/77106962/211050341-1c915c3e-6c2f-4eaa-8de0-51e3d78c4973.png)